modules/packet_network/cloud-init.bash

```diff
@@ -92,6 +92,13 @@ __setup_networking() {
   if ! iptables -C FORWARD -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT; then
     iptables -A FORWARD -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
   fi
+
+  if ! iptables -C INPUT --in-interface "$(__find_local_interface)" --src ! "$(__find_local_subnet)" -j LOG --log-prefix "SPOOF"; then
+    iptables -A INPUT --in-interface "$(__find_local_interface)" --src ! "$(__find_local_subnet)" -j LOG --log-prefix "SPOOF"
+    if ! iptables -C INPUT --in-interface "$(__find_local_interface)" --src ! "$(__find_local_subnet)" -j DROP; then
+      iptables -A INPUT --in-interface "$(__find_local_interface)" --src ! "$(__find_local_subnet)" -j DROP
```

Comment:
Given that the returns from `__find_local_interface` and `__find_local_subnet` are being used multiple times, what do you think about assigning them to local variables?